dev/breeze/src/airflow_breeze/commands/release_management_commands.py

```diff
@@ -821,7 +821,6 @@ def publish_docs(
     "--airflow-site-directory",
     envvar="AIRFLOW_SITE_DIRECTORY",
     help="Local directory path of cloned airflow-site repo.",
-    required=True,
```

Comment:
It's  ok to keep it required I believe + the env var is also providing required value

Reply:
Oh ok, let me try it out with that factor.

Reply:
Yeah seems to work fine. Bringing it back